src/Model/Table/QueuedTasksTable.php

```diff
@@ -3,7 +3,7 @@
 
 use Cake\Core\Configure;
 use Cake\Utility\Hash;
-use Tools\Model\Table\Table;
+use Cake\ORM\Table;
```

Comment:
Why? I use the `'quoteIdentifiers' => true` by default :-)

But I guess it might be useful for some people to not have to depend on it.